bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -37,6 +37,8 @@ struct ClientRequestState {
   std::deque<RequestProcessingStateUniquePtr> reqProcessingHistory;
 };
 
+typedef enum { UNKNOWN_ROLE, PRIMARY, NON_PRIMARY } ReplicaRole;
```

Comment:
Hi @yuliasherman - this is called `ReplicaState` on master and `ReplicaRole` here... Can we align the two branches?